linked_list/linked_list.py

```diff
@@ -40,9 +40,9 @@ def remove_head(self):
       if self.head is None:
         self.tail = None
         
-        return previous
-      else:
-        return None
+      return previous
+    else:
+      return None
     
```

Comment:
Glad you were able to fix this, one tricky thing about python is making sure to indent !